ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/PDEAuxiliaryState.java

```diff
@@ -97,6 +97,7 @@ static class PluginInfo {
 		String localization;
 		String bundleSourceEntry;
 		boolean exportsExternalAnnotations;
+		public String platformFilter;
```

Comment:
I wonder why this is public but the others aren't?

Reply:
That's what happening when using quick-fix to create the field and devloper not checking the results :-)

I just changed this, this might even can become a record these days, but don't want to change to much at once to keep the change count low.